python/ray/data/_internal/execution/streaming_executor.py

```diff
@@ -251,7 +259,7 @@ def _scheduling_loop_step(self, topology: Topology) -> bool:
         """
 
         if DEBUG_TRACE_SCHEDULING:
-            logger.get_logger().info("Scheduling loop step...")
+            logger.get_logger(log_to_stdout=False).info("Scheduling loop step...")
```

Comment:
this one is already behind a debug flag. can keep `log_to_stdout=True` for easier debugging

Reply:
Make sense, changed.